src/server/trpc/router/posts.ts

```diff
@@ -179,6 +181,13 @@ const postsRouter = router({
         })
       }
 
+      // Hidden
+      if (input.hidden !== undefined) {
```

Comment:
I'm unsure about using hidden for deleted.

We need to default it to false here if not provided so that we dont have to add it whenever we want to fetch posts